### DISCUSSION TITLE
Limit aliases to first instance only

### DISCUSSION THREAD

**Discussion by Civilianpig** [SkepticMystic/breadcrumbs#196] [Q&A]
In the matrix/list view, is there a way to limit Breadcrumbs to only show the first alias of notes which have multiple aliases? 

Not sure if this already exists or will be a feature request. There are often times I find it useful to add several aliases to each note to streamline in-text linking with readable formatting. (e.g. different date formats, or different combinations of names and initials). When Breadcrumbs shows every alias separated by commas it adds a lot of clutter to the matrix making it harder to read. By limiting to one, The first alias could be used as the most readable title-version while maintaining full use of the other aliases elsewhere.  

Thanks for a great plugin!

**Comment by SkepticMystic** [comment]
Transferred to an issue here: #237

**Comment by michaelpporter** [comment]
This is now the default behaviour in Breadcrumbs v4. When a view renders a node with its alias, it uses **only the first alias** — the comma-joined list described here no longer happens.

Implemented in the graph engine's node stringifier (`NodeStringifyOptions::stringify_node`): if alias display is enabled and the note has aliases, it renders `aliases.first()`. So your "most readable" first alias shows in the matrix/trail/tree while the remaining aliases stay available for in-text linking — exactly the request.

Tracked as #237 (now closed). Closing this as resolved. 🍞
